tests/unit/test_persist_state.c

```diff
@@ -77,7 +77,8 @@ Test(persist_state, test_persist_state_open_success_on_invalid_file)
   unlink(persist_file);
 
   fd = open(persist_file, O_CREAT | O_RDWR, 0777);
-  write(fd, "aaa", 3);
+  ssize_t ret = write(fd, "aaa", 3);
+  cr_assert_eq(ret, 3, "Write error on invalid persist file");
```

Comment:
Very optional: if we add some error message, can we emit errno as well? Same for the other occurrence below.